### PR TITLE
 removing old styling and resizing events

### PR DESCRIPTION
**Pull request by daleel1**
just removing unnecessary styling

### CODE REVIEW

**Comment by dahfool** [comment]
frontend not a very good name for the branch

**Comment by dahfool** [comment]
image

@reber12 
- The title should say `Events`
- The beginning of the title should be aligned with be begging of the box
- Padding inside the box should be greater, eg 20px
- Spacing between the boxes 20px
- no rounded borders.

File: frontend/src/Components/Event.js [hunk 1]
```diff
@@ -14,7 +12,11 @@ const Event = props => {
             <div>Date: {props.date}</div>
             <div>Floaters: {props.floaters}</div>
             <Popup
-                trigger={<button className="button4"> volunteer</button>}
+                trigger={
+                    <button type="button" class="btn btn-outline-primary">
```
Comment (Behnazz) [inline frontend/src/Components/Event.js]: in react we should use calssName

File: frontend/src/Components/MyForm.js [hunk 2]
```diff
@@ -28,23 +26,46 @@ export default class MyForm extends React.Component {
...
+                </label>
+                <input
+                    className="input form-control"
+                    id="email"
+                    name="email"
+                    type="email"
+                />
+
+                <br />
+                <button type="button" class="btn btn-outline-primary">
```
Comment (Behnazz) [inline frontend/src/Components/MyForm.js]: change class to className

**Comment by dahfool** [comment]
@reber12 Great, i have seen the changes we need some spacing between the button and text as well. You can use bootstrap spacing for that - https://getbootstrap.com/docs/4.1/utilities/spacing/

Add this class to the button `mt-4`

**Review by dahfool — APPROVED** [review]

File: frontend/src/Components/Event.js [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
             <h1>{props.name}</h1>
             <p>Description: {props.description}</p>
             <div>Date: {props.date}</div>
             <div>Floaters: {props.floaters}</div>
-            <Popup
-                trigger={<button className="button4"> volunteer</button>}
-                position="right center"
-                modal
-            >
+            <Popup trigger={<button type="button" class="btn btn-outline-primary mt-4">
```
Comment (anton-kozyr) [inline frontend/src/Components/Event.js]: use ``className``

**Review by anton-kozyr — CHANGES_REQUESTED** [review]
replace ```class``` with ```className```

**Comment by dahfool** [comment]
@reber12 you have some conflicts.

**Comment by daleel1** [comment]
> @reber12 you have some conflicts.

@dahfool sorry I can not spot conflicts

**Review by anton-kozyr — APPROVED** [review]

**Pull request merged**